because user namespaces have been disabled
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Fri Apr  5 20:31:46 2024... 
[Fri Apr  5 20:52:37 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Mon Apr  8 15:09:13 2024... 
[Mon Apr  8 15:27:41 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Mon Apr  8 16:20:28 2024... 
[Mon Apr  8 16:38:41 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 330, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Mon Apr  8 19:23:08 2024... 
[Mon Apr  8 19:46:06 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Wed Apr 10 06:37:32 2024... 
[Wed Apr 10 06:58:12 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Wed Apr 10 17:33:21 2024... 
[Wed Apr 10 17:55:02 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Wed Apr 10 19:20:29 2024... 
[Wed Apr 10 19:39:26 2024]  Iteration number: 0 with current cost as 0.36574513509654577 and parameters 
[-4.34861199  2.23743429 -2.12427861 -0.1165324   0.55388502 -2.77011
  3.06858121  2.18960368  1.18551827 -1.06648274  0.60271304  1.14432325
  1.3102995  -1.87354646  0.72964806  2.88578539 -0.54534335 -0.47522519
 -2.02654412  0.72897404  1.60512629  2.83077433 -1.26456727 -0.25136293
 -2.39279372 -2.27309843  3.13337018  2.54856873 -0.67550959 -2.6900227
  1.51701451 -0.04647263 -2.34271679  2.04043403  1.15216415 -0.65055327
  1.04464957 -1.9205605  -0.32068348  2.32225879 -0.20585058  0.56636987
  2.68983097  1.07273276  2.58102901  2.94254101  2.76514642 -2.94578365
  0.36684025 -0.65725924  1.65158898  2.68041884 -0.25194851 -2.45807666
  2.63570542 -3.07930429  0.79129724 -0.75901771 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Training complete taking 1715.3477110862732 total seconds. 
Now scoring model... 
Scoring complete taking 9.770401954650879 seconds. 
Saved predicted values as A2-A2-CZ_Modified-Pauli-CRZ_predicted_values.csv
Model scores: {'MSE_train': (228.77309839163405,), 'R2_train': -0.10832888181393407, 'MAE_train': 13.468675385748952, 'MSE_test': 200.4137962106918, 'R2_test': -0.2080570685627796, 'MAE_test': 11.976916783568488}. 
Saved model results as A2-A2-CZ_Modified-Pauli-CRZ_results.json. 
